app/views/layouts/application.html.erb

```diff
@@ -69,6 +69,19 @@
 		</div><!--/.container-fluid -->
 	</div>
 
+	<% if notice %>
+	<br/>
+	<br/>
+	<p class="alert alert-success"><%= notice %></p>
+	<% notice = '' %>
```

Comment:
This is not required. Rails will automatically do this once it gets rendered.

Remove these lines in notice and alert.